src/Caching/SqlServer/src/Columns.cs

```diff
@@ -22,7 +22,7 @@ public static class Indexes
             public const int ExpiresAtTimeIndex = 1;
             public const int SlidingExpirationInSecondsIndex = 2;
             public const int AbsoluteExpirationIndex = 3;
```

Comment:
Are these three constants needed now that they aren't included the in the query at all?

Reply:
Indeed, my bad, I should realize that these constants are no longer needed. In fact, the constant at the line 21 is not needed either. I'll remove them in a commit.

The only constant that would remain is `CacheItemValueIndex`. Even that is the only one that remains, I'm in favor to keep it because adds some level of consistency and makes easier to track and refactor the code. Just finding the references is possible to know where this column is used.